libcxx/include/__ranges/iota_view.h

```diff
@@ -393,6 +393,15 @@ struct __fn {
 inline namespace __cpo {
 inline constexpr auto iota = __iota::__fn{};
 } // namespace __cpo
+
+#  if _LIBCPP_STD_VER >= 26
+
+inline constexpr auto indices = [](__integer_like auto __size) static {
```

Comment:
I don't think libc++ provides any (cv-unqualified) `__integer_like` but not `weakly_incrementable` type at this moment. So, no change requested at this moment. Although the `operator()` will need to be constrained harder once libc++ starts to provide such an integer-class type.